src/main/resources/application.yml

```
@@ -17,6 +17,17 @@ spring:
       fail-on-empty-beans: false
   main:
     allow-bean-definition-overriding: true
+  flyway:
+    enabled: true
+  datasource:
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
```

Jeg hadde sagt feil her, det var tydligen DB_DATABASE. Men sikkert fint å legge til NOT NULL i databasen før denne legges inn :) 
```suggestion
    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_DATABASE}
```